### ISSUE TITLE
The announcer report is too long 

### ISSUE THREAD

**Issue by ofekashery** [FIRST-Tech-Challenge/scorekeeper#339]
It should be much shorter - about 100 pages per event is unlikely.
Maybe something like this:
#### Before:
![before](https://user-images.githubusercontent.com/16443111/52123639-6d43f780-262f-11e9-8774-e74a7c2d1ae7.png)
-----------
#### After:
![after](https://user-images.githubusercontent.com/16443111/52123638-6cab6100-262f-11e9-8089-d2691764bff5.png)

**Comment by Nate2019** [comment]
I disagree.... Just use a tablet for the announcer report and it works perfectly.

**Comment by ofekashery** [comment]
This table is fine, I just wanted to show that it can be reduced by half

**Comment by maths22** [comment]
The announcer report has been redesigned
